hal/src/sercom/i2c.rs

```diff
@@ -276,6 +276,9 @@ pub use config::*;
 
 mod impl_ehal;
 
+// mod client;
```

Comment:
I would try to standardize what terminology we use: master/slave vs host/client. Personally I don't care either way, with maybe a slight preference towards whatever the datasheet uses.

Reply:
Thanks for bringing this up, I meant to ask. The datasheet does use host/client, so I can go ahead with that.